### PR TITLE
dev: add covid alert component

### LINKED ISSUE #10: Add COVID banner component
Create a component for rendering the COVID alert: https://github.com/USSF-ORBIT/ussf-portal-mvp/blob/main/_includes/banner-alert.html

### PR DESCRIPTION
**Pull request by abbyoung**
## Description 

* Adds covid alert component 
* Displays component in Storybook
* Adds tests

Fixes #10 

## Review Notes

* We may need to fix FontAwesome to use the Pro icons
  * Only the standard weight for the arrowRight icon is available in the free pack
  * @noahfirth Is there a Pro license being used in the existing MVP we can/should use?
  
Screen Shot 2021-07-13 at 5 43 38 PM
Screen Shot 2021-07-13 at 5 43 30 PM

### CODE REVIEW

File: components/MVP/CovidSiteAlert/CovidSiteAlert.module.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+@import 'styles/uswdsDependencies';
+@import 'styles/mvp/mixins';
+
+.usa-site-alert--info {
```
Comment (suzubara) [inline components/MVP/CovidSiteAlert/CovidSiteAlert.module.scss:4]: In order to not confuse local CSS module class names with global/USWDS classnames, I'd suggest using a different naming convention, maybe `.covidSiteAlert`?

**Review by suzubara — COMMENTED** [review]
this looks good! Just a suggestion around naming CSS module classes

**Review by suzubara — APPROVED** [review]
🎉

File: components/MVP/CovidSiteAlert/CovidSiteAlert.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+const CovidSiteAlert = () => {
+  return (
+    <SiteAlert
+      variant="info"
+      showIcon={false}
+      className={`${styles.covidSiteAlert}`}>
+      <FontAwesomeIcon
+        // The icon in the MVP uses the fal prefix/light variation.
+        // It requires a Pro license -- do we have this/need this?
```
[the remark this answers is not in the document]
Comment (noahfirth) [reply components/MVP/CovidSiteAlert/CovidSiteAlert.tsx:15]: Yep, we're set for now.

**Review by suzubara — APPROVED** [review]

**Pull request merged**